requirements.txt

```diff
@@ -1,2 +1,3 @@
 BeautifulSoup==3.2.1
 lxml==3.4.4
```

Comment:
Woah, this isn't related to review, but I wonder why we need both BeautifulSoup and lxml..